### PR TITLE
Fix hasBlock deprecation

### PR DESCRIPTION
**Pull request by renaudleo**
`hasBlock` is deprecated in favor of `has-block` and will break in Ember 4. Building an app that uses `ember-place-autocomplete` results in the following warning on recent Ember versions:

> DEPRECATION: `hasBlock` is deprecated. Use `has-block` instead. ('ember-place-autocomplete/templates/components/place-autocomplete-field.hbs' @ L1:C6)

See the following references:

- https://deprecations.emberjs.com/v3.x/#toc_has-block-and-has-block-params
- https://github.com/emberjs/rfcs/blob/master/text/0689-deprecate-has-block.md

### CODE REVIEW

**Comment by navels** [comment]
ping @dmuneras

**Review by navels — APPROVED** [review]

**Comment by cloutierlp** [comment]
Friendly ping in order to get this PR merged 🥇

**Review by esbanarango — APPROVED** [review]

**Comment by navels** [comment]
> Friendly ping in order to get this approved PR merged 🥇

I reached out to @dmuneras on LinkedIn a few months ago, he was going to make me a collaborator which would unblock this.  I just pinged him again.

**Comment by dmuneras** [comment]
> > Friendly ping in order to get this approved PR merged 🥇
> 
> I reached out to @dmuneras on LinkedIn a few months ago, he was going to make me a collaborator which would unblock this. I just pinged him again.

Hi man, 

I made you collaborator few months ago. 

FEFD697C-C2AC-47C1-9FEA-999C50E95D2A

 Let me know if you are missing any permission or if you missed the invitation.

**Comment by navels** [comment]
@dmuneras thanks!  I don't recall (and can't find in my email) an invitation . . . not sure if that matters. But I can't merge this PR, but possibly that is because CI isn't running . . . not sure.

**Comment by dmuneras** [comment]
> @dmuneras thanks! I don't recall (and can't find in my email) an invitation . . . not sure if that matters. But I can't merge this PR, but possibly that is because CI isn't running . . . not sure.

Cool cool! I am not actively working on Ember since a while ago, therefore I decided to transfer the repo to @esbanarango. Once I have the luck to get back I will gladly contribute to it again.

**Review by esbanarango — APPROVED** [review]

**Comment by esbanarango** [comment]
I'll be releasing a new version this week and starting the work to migrate this repo into a V2 addon. (And moving away from Travis)

**Pull request merged**
